core/templates/dev/head/components/share/sharing_links_directive.html

```diff
@@ -2,25 +2,25 @@
   <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
     <li>
       <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareExplorationEvent('gplus')" target="_window">
-        <img class="share-option-img" ng-src="<[gplusUrl]>" alt="Google+">
+        <img class="oppia-hide-mobile share-option-img" ng-src="<[gplusUrl]>" alt="Google+">
```

Comment:
I think this is the wrong place to do this. Can you just hide the entire `<sharing-links>` directive on mobile and leave this file unchanged?